### PR TITLE
added the may newsletter

### PR DESCRIPTION
**Pull request by FlanFlanagan**
first draft!

### CODE REVIEW

File: source/newsletters/may2018.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+.. raw:: html
+
+    </div>
+
+
+Greetings! Warm weather is upon us and the cyclus development team is looking forward to a productive summer. This past fall and spring have been busy for us! But we still have some wonderful things to share. 
+
+Developments
+============
+Cyclus has been undergoing a number of changes recently. These changes have not been released in a full version as of yet. However, as they are forth coming the following is a list of changes coming with the next release.
```
Comment (katyhuff) [inline source/newsletters/may2018.rst]: forthcoming is one word, I think.

File: source/newsletters/may2018.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+.. image:: ../logos/logo2_bike.gif
+    :align: center
+    :alt: Cyclus
+
+.. raw:: html
+
+    </div>
+
+
+Greetings! Warm weather is upon us and the cyclus development team is looking forward to a productive summer. This past fall and spring have been busy for us! But we still have some wonderful things to share.
```
Comment (katyhuff) [inline source/newsletters/may2018.rst]: Capitalize Cyclus. Not sure if the third sentence should start with But. Probably "Accordingly" ?

File: source/newsletters/may2018.rst (hunk 1 above)
Comment (FlanFlanagan) [reply source/newsletters/may2018.rst]: Thanks! I'll take care of it!

File: source/newsletters/may2018.rst (hunk 2 above)
Comment (FlanFlanagan) [reply source/newsletters/may2018.rst]: Good catch! Thanks!

**Pull request merged**